torchao/quantization/quant_primitives.py

```diff
@@ -205,15 +203,16 @@ def quantize_affine(
 
     return quant
 
+@register_custom_op("quant::dequantize_affine")
```

Comment:
does this make the default behavior to preserve the higher level op when we run export? What about compile?

Reply:
yes, and in compile we'll register a inductor decomposition to decompose the op